### ISSUE TITLE
`win_get_id` and `win_get_idlast` should be returning integers

### ISSUE THREAD

**Issue by trajano** [spyoungtech/ahk#325]
### describe your issue

* https://www.autohotkey.com/docs/v2/lib/WinGetID.htm
* https://www.autohotkey.com/docs/v2/lib/WinGetIDLast.htm

Returns an integer which represents the hwnd

but ahk returns a `Window` object. 

Ideally there shouldn't be a need for those specific methods provided ahk.win_get provides the HWND, unfortunately the `id` in AHK's Window isn't the hwnd, but it appears to be `ahk_id` which isn't an HWnd either.

### AutoHotkey version

v2

### Traceback/Error message

x

**Comment by spyoungtech** [comment]
For the same reasons discussed in #269 the ID is a string. Short version: in AHKv1 (against which the wrapper was designed first) they are strings. When v2 support was added, the return value was kept as a string for type concurrency between v1 and v2. That makes it easier for users using static type checkers to not need to know the AHK version ahead of time (which may only be determined at runtime, in some cases).

**Comment by spyoungtech** [comment]
`Window` objects are basically just a container around the window ID. The window ID is its hwnd. The value is exactly the same as the window's `id` property. In fact, `WinGetId` is [used in the implementation](https://github.com/spyoungtech/ahk/blob/86d9e56308498a68f75499c842df74085d2a9287/ahk/templates/daemon-v2.ahk#L576).

The reason why a `Window` object is returned is because that's almost always what you want to do with an hwnd anyhow. If you really wanted the raw hwnd value for some reason, you can access it with the `id` property.

Potentially related: #269

**Comment by trajano** [comment]
OK the HWND that it may be providing is liekly not a window handle.

```
    if not user32.MoveWindow(window.id, x, y, 1920, 2100, True):
        raise ctypes.WinError(ctypes.get_last_error())
```

results in

> OSError: [WinError 1400] Invalid window handle.

A code I wrote to locate the possible window handles and check the value of `window.id`

```python
from ahk import AHK
import  pywinctl as pwc
from ahk.directives import NoTrayIcon
import pygetwindow as gw
import ctypes
from ctypes import wintypes

one_note_ahk_title = "ahk_exe ONENOTE.EXE"
user32 = ctypes.WinDLL('user32', use_last_error=True)
window =ahk.win_get(one_note_ahk_title)
result = list(map(lambda it: it._hWnd, pwc.getAllWindows()))

print(result)
print(window.id)

entry = window.id
if entry in result:
    print(f"{entry} is in the list.")
else:
    print(f"{entry} is not in the list.")
```

**Comment by spyoungtech** [comment]
I think this is resolved in the discussion #324 so I will close this for now. But feel free to reopen if you have other issues.
